README.md

```diff
@@ -2,20 +2,30 @@
 services: active-directory
 platforms: dotnet
 author: jmprieur
+level: 200
+client: UWP 
+service: Microsoft Graph
+endpoint: AAD V1
 ---
+# Universal Windows Platform (UWP) application calling the Microsoft Graph using Windows 10 Web Account Manager
 
-# Universal Windows Platform (UWP) app calling the directory Graph API
+![Build badge](https://identitydivision.visualstudio.com/_apis/public/build/definitions/a7934fdd-dcde-4492-a406-7fad6ac00e17/515/badge)
 
-This sample demonstrates a Universal Windows Platform (UWP) app calling the directory Graph API to look up a user. The UWP app uses the Windows 10 WebAccountManager API to obtain an access token for the Graph as the currently signed in user, or any valid Azure AD account entered by the user.
+This sample demonstrates a Universal Windows Platform (UWP) app calling the directory Graph API to look up a user. The UWP app uses the Windows 10 [WebAccountManager API](https://docs.microsoft.com/en-us/uwp/api/Windows.Security.Authentication.Web.Provider.WebAccountManager) to obtain an access token for the Graph as the currently signed in user, or any valid Azure AD account entered by the user.

```

Comment:
Changing the reference to be to the conceptual article. Adding the reference as a "More information" link at the bottom of the article